server/controllers/DocumentControllers.js

```diff
@@ -0,0 +1,215 @@
+import pagination from '../helper/pagination';
+
+const Document = require('../models').Document;
+const User = require('../models').User;
+
+const DocumentControllers = {
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns a response object.
+   */
+  create(req, res) {
+    const authorId = res.locals.decoded.UserId;
+    return Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      UserId: authorId,
+      access: req.body.access
+    })
+    .then(document => res.status(201).send({
+      message: 'Document successfully created',
+      document,
+    })
+    )
+    .catch(() => res.status(400).send({
+      message: 'Please verify your input'
+    }));
+  },
+
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns the list of documents available
+   */
+
+  listDocuments(req, res) {
+    const limit = req.query.limit || 10;
+    const offset = req.query.offset || 0;
+
+    if (isNaN(limit) || isNaN(offset)) {
+      return res.status(400).send({
+        message: 'limit and offset must be an integer'
+      });
+    }
+    if (req.query) {
+      return Document.findAndCount({
+        limit,
+        offset,
+      })
+    .then(document => res.status(200).send({
+      pagination: {
+        row: document.rows,
+        paginationDetails: pagination(document.count, limit, offset)
+      }
+    }))
+    .catch(() => res.status(403).send({
+      message: 'limits and offsets must be number'
+    }));
+    }
+  },
+
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} returns the requested document
+   */
+  getDocument(req, res) {
+    if (isNaN(req.params.id)) {
+      return res.status(400).send({
+        message: 'Id must be an integer'
+      });
+    }
+    return User
+    .findById(res.locals.decoded.UserId, {
```

Comment:
Why did you break this into another line. would be easier to read if you keep `User.findById` on same line